### DISCUSSION TITLE
I have updated my Docusaurus to v3.0, but the size of the buttons in the document has changed

### DISCUSSION THREAD

**Discussion by SilianZ** [facebook/docusaurus#9534] [Q&A] [answered]
我将我的 Docusaurus 文档更新到了 v3.0，但是文档中按钮的大小发生了变化。

![image](https://github.com/facebook/docusaurus/assets/113701655/6eb8772d-0b1c-4ec7-9641-c240fdc208f3)

Just like this, the text displayed in the button is not in the center of the button.

如图，按钮中的文本并没有居中。

Here is my code.

这是我的代码。

```mdx
import Link from '@docusaurus/Link';
import styles from './index.module.css';
import HomepageFeatures from '@site/src/components/HomepageFeatures';

<div align="center">
    <img width="256" src="./character_marked.png" alt="logo" />
    <p>（该形象版权归 Teahouse-Studios 所有）</p>

# 小可 · AkariBot v4

</div>
<div className={styles.buttons}>
    <Link
        className="button button--primary button--lg"
        to="/docs/intro">
        开始使用 ⏱️
    </Link>
</div>
<HomepageFeatures />
```

**Comment by slorber** [comment]
Please create a repro and show the before/after upgrade screenshot.

Also I don't speak your language sorry 😅

**Reply by SilianZ** [reply]
Before:
![image](https://github.com/facebook/docusaurus/assets/113701655/3f814a1f-cf35-49a4-bbc4-4b7c4247913c)
https://akari-site-88nfnby4n-silianz.vercel.app/
After:
![image](https://github.com/facebook/docusaurus/assets/113701655/4d7f0184-8df4-4926-abe9-ada0225df0b8)
https://akari-site-dgyak7dqj-silianz.vercel.app/

**Reply by SilianZ** [reply, marked as answer]
bruh the problem has been solved:
```mdx
<div className={styles.buttons}>
    <Link
        className="button button--secondary button--lg"
        to="/docs/intro">
        开始使用 >
    </Link>
</div>
```
↓↓↓
```mdx
<div className={styles.buttons}>
    <Link className="button button--secondary button--lg" to="/docs/intro">开始使用 ></Link>
</div>
```
